docs/troubleshooting.md

```diff
@@ -108,19 +115,19 @@ If everything seems healthy, and still no packages show up in the UI, it could b
 
 
 
-## OperatorSource failing to download from datastore
+## OperatorSource failing to connect to the datastore
 
-If an OperatorSource fails to reconcile for any reason (i.e operators in that particular OperatorSource fail to show up in the UI), it might be helpful first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator for detailed information about the OperatorSource. 
+If an OperatorSource fails to reconcile for any reason, first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator with `oc logs <name-of-marketplace-operator> -n openshift-marketplace` for detailed information about the OperatorSource. 
 
-Note: A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [end point](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the end point is specified as `https://quay.io/cnr`.
+A common reason for an OperatorSource to not reconcile correctly has been the use of the wrong [endpoint](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/examples/community.operatorsource.cr.yaml#L8). Please make sure that the endpoint is specified as `https://quay.io/cnr`.
```

Comment:
`not reconcile correctly is the use of`

Please use consistent tense. Present tense generally applies when making declarative statements. The use of the present perfect here (https://www.ef.edu/english-resources/english-grammar/present-perfect/) can sometimes have accidental negative connotation.